doc/sphinxext/gen_rst.py

```diff
@@ -67,12 +67,11 @@ def execfile(filename, global_vars=None, local_vars=None):
     import matplotlib
```

Comment:
Nice, didn't know that existed. I assume you mean in a subsequent PR of course :)